custom_components/kamstrup_403/sensor.py

```diff
@@ -29,6 +29,14 @@
         device_class=SensorDeviceClass.ENERGY,
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    SensorEntityDescription(
+        key="63",  # 0x003F
+        name="Cooling Energy (E3)",
+        icon="mdi:weather-windy",
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+	entity_registry_enabled_default=False,
```

Comment:
```suggestion
        entity_registry_enabled_default=False,
```
Use spaces instead of tab

Reply:
Good to know, its corrected.